opba-db/src/test/java/de/adorsys/opba/db/CsvConvertTest.java

```diff
@@ -0,0 +1,55 @@
+package de.adorsys.opba.db;
+
+import com.google.common.io.Files;
+import com.google.common.io.Resources;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Disabled
+public class CsvConvertTest {
+    private static final String BANKS_SOURCE = "migration/migrations/banks.csv";
+    private static final String BANKS_ACTION_SQL = "bank_action_sql.txt";
+    private static final String BANKS_SUB_ACTION_SQL = "bank_sub_action_sql.txt";
+
+    @Test
+    @SneakyThrows
+    public void convertToDbCsv() {
+        int bankId = 1;
+        int bankActionId = 1;
+        List<String> banks = readResourcLines(BANKS_SOURCE);
+        banks.remove(0);
+
+        for (String bank : banks) {
+            int authorizationId;
+            String[] data = bank.split(",");
+            writeToFile(BANKS_ACTION_SQL, "insert into ${table-prefix}bank_action (id, bank_uuid, protocol_action, protocol_bean_name) values (" + bankId++ + ", '" + data[0] + "', 'LIST_ACCOUNTS', 'xs2aListAccounts');");
```

Comment:
Can we use String.format instead of concatenations?

Reply:
String.format was used on every concatenation